templates/scenarios/notification-restify-js/.appserviceIgnore

```diff
@@ -12,6 +12,7 @@ local.settings.json
 test
 tsconfig.json
 .DS_Store
+node_modules
```

Comment:
We need to keep the node modules folder in package, cause RUN_FROM_PACKAGE needs all dependencies

Reply:
thanks, resolved